packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md

```diff
@@ -4,13 +4,94 @@ title: Okta Identity Engine API Products release notes 2023
 
 <ApiLifecycle access="ie" />
 
+## March
+
+### Monthly release 2023.03.0
+
+| Change | Expected in Preview Orgs |
+|---------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------|
+| [Authenticator enrollment using the /authorize endpoint is GA in Production](#authenticator-enrollment-using-the-authorize-endpoint-is-ga-in-production)       |March 06, 2023   |
+| [OIDC Identity Providers private/public key pair support is GA](#oidc-identity-providers-private-public-key-pair-support-is-ga)                                |March 06, 2023   |
+| [API service integrations are GA in Preview](#api-service-integrations-are-ga-in-preview)                                                                      |March 06, 2023   |
+| [Log Streaming is GA in Production](#log-streaming-is-ga-in-production)                                                                                        |March 06, 2023   |
+| [Optional consent for OAuth 2.0 scopes is GA in Prod](#optional-consent-for-oauth-2-0-scopes-is-ga-in-prod)                                                    |March 06, 2023   |
+| [OAuth 2.0 authentication for inline hooks is GA in Preview](#oauth-2-0-authentication-for-inline-hooks-is-ga-in-preview)                                      |March 06, 2023   |
+| [Transactional verification with CIBA is GA in Preview](#transactional-verification-with-ciba-is-ga-in-preview)                                                |March 06, 2023   |
+| [Improvements to self-service account activities for AD and LDAP users](#improvements-to-self-service-account-activities-for-ad-and-ldap-users)                |March 06, 2023   |
+| [Honor force authentication support for SAML apps API](#honor-force-authentication-support-for-saml-apps-api)                                                  |March 06, 2023   |
+| [OIN Manager support for Workflow Connector submission is GA in Preview](#oin-manager-support-for-workflow-connector-submission-is-ga-in-preview)              |March 06, 2023   |
+| [Rate limit increased for Event Hooks](#rate-limit-increased-for-event-hooks)                                                                                  |March 06, 2023   |
+| [Bugs fixed in 2023.03.0](#bugs-fixed-in-2023-03-0)                                                                                                           |March 06, 2023   |
```

Comment:
```suggestion
| [Bugs fixed in 2023.03.0](#bugs-fixed-in-2023-03-0)                                                                                                           |March 08, 2023   |
```
Monthly is scheduled for deployment on March 8th. Please update all appropriate dates.